client/extensions/wp-job-manager/components/setup/page-setup/index.jsx

```diff
@@ -0,0 +1,156 @@
+/**
+ * External dependencies
+ */
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import Button from 'components/button';
+import Card from 'components/card';
+import CompactFormToggle from 'components/forms/form-toggle/compact';
+import ExternalLink from 'components/external-link';
+import FormFieldset from 'components/forms/form-fieldset';
+import FormSettingExplanation from 'components/forms/form-setting-explanation';
+import FormTextInput from 'components/forms/form-text-input';
+import SectionHeader from 'components/section-header';
+
+class PageSetup extends Component {
+	static propTypes = {
+		translate: PropTypes.func.isRequired,
+	};
+
+	state = {
+		dashboard: true,
+		dashboardPage: this.props.translate( 'Job Dashboard' ),
+		jobs: true,
+		jobsPage: this.props.translate( 'Jobs' ),
```

Comment:
Hi! I've found a possible matching string that has already been translated **5** times:
`translate( 'Jobs', { context: 'jobs'} )` *ES Score: 15*
See 1 additional suggestion in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/wpjm-wizard-page-setup)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).